model/agent.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Agent",
+  "description": "An Elastic Agent that has enrolled into Fleet",
+  "type": "object",
+  "properties": {
+    "_id": {
+      "description": "The unique identifier for the Elastic Agent",
+      "type": "string",
+      "format": "uuid"
+    },
+    "_version": {
+      "description": "The version of the document in the index",
+      "type": "integer"
+    },
+    "agent_version": {
+      "description": "The version of the Elastic Agent",
+      "type": "string"
+    },
+    "local_metadata": {
```

Comment:
How do you model the second level fields with this structure?

Reply:
@nchaulet Are these modeled or just free-form fields?

We could change to model them if they don't change.

Reply:
These models were changing a lot during the first iterations of Fleet, but we can probably have a fixed model now.